src/TestInfrastructure/Helpers/MefTestHelpers.cs

```diff
@@ -120,18 +120,37 @@ public static Export CreateExport<T>(object exportInstance, string contractName
         /// <param name="typeToExport">The type to of export being tested</param>
         /// <param name="additionalExports">Any other exports that are required. Can be empty.</param>
         public static void Compose(object importer, Type typeToExport, params Export[] additionalExports)
+            => Compose(
+                new object[]{ importer },
+                new Type[]{ typeToExport },
+                additionalExports);
+
+        /// <summary>
+        /// Performs a MEF compose with the supplied objects
+        /// </summary>
+        /// <param name="importers">The instance that is importing the exported type</param>
+        /// <param name="typesToExport">The types of export being tested</param>
+        /// <param name="additionalExports">Any other exports that are required. Can be empty.</param>
+        /// <remarks>Note: this overload is useful for testing types that have multiple exports
+        /// i.e. that export multiple types.</remarks>
+        public static void Compose(object[] importers, Type[] typesToExport, params Export[] additionalExports)
```

Comment:
Added an overload of the existing method that accepts arrays for all three arguments.